2024-07-10 22:50:08,084 INFO: Dieta criada com sucesso: {'id': 15, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []} [in /app/app.py:39]
2024-07-10 22:50:08,133 INFO: 172.30.0.1 - - [10/Jul/2024 22:50:08] "POST /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:50:15,015 INFO: Listagem de dietas realizada [in /app/app.py:54]
2024-07-10 22:50:15,018 INFO: 172.30.0.1 - - [10/Jul/2024 22:50:15] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:52:12,001 INFO: Logging setup complete [in /app/logs/log_config.py:25]
2024-07-10 22:52:12,101 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://172.30.0.3:5001 [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:52:12,102 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:52:15,277 INFO: Listagem de dietas realizada: [{'id': 15, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []}] [in /app/app.py:55]
2024-07-10 22:52:15,279 INFO: 172.30.0.1 - - [10/Jul/2024 22:52:15] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:53:48,786 INFO: Logging setup complete [in /app/logs/log_config.py:25]
2024-07-10 22:53:48,844 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.32.3:5001 [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:53:48,845 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:18,395 INFO: Listagem de dietas realizada: [{'id': 15, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []}] [in /app/app.py:55]
2024-07-10 22:54:18,396 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:18] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:19,177 INFO: Listagem de dietas realizada: [{'id': 15, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []}] [in /app/app.py:55]
2024-07-10 22:54:19,178 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:19] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:31,242 INFO: Dieta com ID 15 deletada com sucesso. [in /app/app.py:71]
2024-07-10 22:54:31,244 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:31] "DELETE /dietas/15 HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:34,578 INFO: Listagem de dietas realizada: [] [in /app/app.py:55]
2024-07-10 22:54:34,580 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:34] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:36,293 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:36] "OPTIONS /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:36,309 INFO: Dieta criada com sucesso: {'id': 16, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []} [in /app/app.py:39]
2024-07-10 22:54:36,310 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:36] "POST /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:54:39,112 INFO: Listagem de dietas realizada: [{'id': 16, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []}] [in /app/app.py:55]
2024-07-10 22:54:39,113 INFO: 192.168.32.1 - - [10/Jul/2024 22:54:39] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:57:03,637 INFO: Dieta com ID 16 deletada com sucesso. [in /app/app.py:71]
2024-07-10 22:57:03,639 INFO: 192.168.32.1 - - [10/Jul/2024 22:57:03] "DELETE /dietas/16 HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:57:15,422 INFO: 192.168.32.1 - - [10/Jul/2024 22:57:15] "OPTIONS /horarios_refeicoes HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:57:15,429 ERROR: Erro de valida√ß√£o ao criar hor√°rio de refei√ß√£o: [{'loc': ('tipo',), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('refeicoes',), 'msg': 'field required', 'type': 'value_error.missing'}] [in /app/app.py:86]
2024-07-10 22:57:15,431 INFO: 192.168.32.1 - - [10/Jul/2024 22:57:15] "[31m[1mPOST /horarios_refeicoes HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:57:21,263 INFO: 192.168.32.1 - - [10/Jul/2024 22:57:21] "OPTIONS /horarios_refeicoes HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 22:57:21,269 ERROR: Erro de valida√ß√£o ao criar hor√°rio de refei√ß√£o: [{'loc': ('tipo',), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('refeicoes',), 'msg': 'field required', 'type': 'value_error.missing'}] [in /app/app.py:86]
2024-07-10 22:57:21,270 INFO: 192.168.32.1 - - [10/Jul/2024 22:57:21] "[31m[1mPOST /horarios_refeicoes HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:01:53,157 INFO: Logging setup complete [in /app/logs/log_config.py:25]
2024-07-10 23:01:53,215 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.48.3:5001 [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:01:53,216 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:00,188 INFO: Listagem de dietas realizada: [] [in /app/app.py:76]
2024-07-10 23:02:00,190 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:00] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:04,214 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:04] "OPTIONS /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:04,232 ERROR: Erro desconhecido ao criar dieta: 'DietaCreate' object has no attribute 'horarios_refeicoes' [in /app/app.py:67]
2024-07-10 23:02:04,234 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:04] "[35m[1mPOST /dietas HTTP/1.1[0m" 500 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:05,717 ERROR: Erro de integridade ao criar dieta: (pymysql.err.IntegrityError) (1062, "Duplicate entry 'V5nmLXsJlJRC5lvs53TxOnFPvPv1' for key 'dietas.firebase_uuid'")
[SQL: INSERT INTO dietas (firebase_uuid) VALUES (%(firebase_uuid)s)]
[parameters: {'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1'}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /app/app.py:63]
2024-07-10 23:02:05,718 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:05] "[35m[1mPOST /dietas HTTP/1.1[0m" 500 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:08,526 INFO: Listagem de dietas realizada: [{'id': 17, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []}] [in /app/app.py:76]
2024-07-10 23:02:08,527 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:08] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:16,560 INFO: Dieta com ID 17 deletada com sucesso. [in /app/app.py:92]
2024-07-10 23:02:16,561 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:16] "DELETE /dietas/17 HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:19,545 INFO: Listagem de dietas realizada: [] [in /app/app.py:76]
2024-07-10 23:02:19,547 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:19] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:20,926 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:20] "OPTIONS /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:20,940 ERROR: Erro desconhecido ao criar dieta: 'DietaCreate' object has no attribute 'horarios_refeicoes' [in /app/app.py:67]
2024-07-10 23:02:20,942 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:20] "[35m[1mPOST /dietas HTTP/1.1[0m" 500 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:02:23,481 INFO: Listagem de dietas realizada: [{'id': 19, 'firebase_uuid': 'V5nmLXsJlJRC5lvs53TxOnFPvPv1', 'horarios_refeicoes': [], 'refeicoes': []}] [in /app/app.py:76]
2024-07-10 23:02:23,482 INFO: 192.168.48.1 - - [10/Jul/2024 23:02:23] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:03:45,248 INFO: Dieta com ID 19 deletada com sucesso. [in /app/app.py:92]
2024-07-10 23:03:45,250 INFO: 192.168.48.1 - - [10/Jul/2024 23:03:45] "DELETE /dietas/19 HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:03:49,240 INFO: Listagem de dietas realizada: [] [in /app/app.py:76]
2024-07-10 23:03:49,241 INFO: 192.168.48.1 - - [10/Jul/2024 23:03:49] "GET /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:03:52,120 INFO: 192.168.48.1 - - [10/Jul/2024 23:03:52] "OPTIONS /dietas HTTP/1.1" 200 - [in /usr/local/lib/python3.10/site-packages/werkzeug/_internal.py:187]
2024-07-10 23:03:52,133 ERROR: Erro desconhecido ao criar dieta: 'DietaCreate' object has no attribute 'horarios_refeicoes' [in /app/app.py:67]
